web/src/components/storage/ProposalFailedInfo.tsx

```diff
@@ -86,8 +88,11 @@ const Description = () => {
  *   - The generated proposal contains no errors.
  */
 export default function ProposalFailedInfo() {
-  const issues = useScopeIssues("storage");
-  if (issues.length === 0) return;
+  const configErrors = useConfigIssues();
+  const errors = useIssues("storage");
```

Comment:
to be honest I do not get rename of issues to errors here. It is a bit confusing. I would probably keep it in original form `configIssues` and `issues`